### PR TITLE
Feature/set

### PR DESCRIPTION
**Pull request by Skrikunov**
OK

### CODE REVIEW

File: storage.py [hunk 1]
```diff
@@ -15,8 +15,9 @@ def get(self, key):
     def remove(self):
         pass
 
-    def set(self):
-        pass
-    
+    def set(self, key, value):
+        if key in self.data:
+            self.data[key] = value
```
Comment (kmisterios) [inline storage.py:20]: If there is no key in the data, the function set should return `None`, I suppose.

File: storage.py [hunk 2]
```diff
@@ -15,8 +15,9 @@ def get(self, key):
     def remove(self):
         pass
 
-    def set(self):
-        pass
-    
+    def set(self, key, value):
+        if key in self.data:
```
Comment (kmisterios) [inline storage.py:19]: To access the array of keys you should write `self.data.keys`. Does the current version actually work?

File: storage.py (hunk 1 above)
Comment (Skrikunov) [reply storage.py:20]: I agree with you. I'll fix it.

File: storage.py (hunk 2 above)
Comment (Skrikunov) [reply storage.py:19]: Yes, moreover, such a construction is used in the get() implementation.

Comment (kmisterios) [reply storage.py:19]: Ok, I haven't noticed it)

File: storage.py (hunk 1 above)
Comment (maxkuzn) [reply storage.py:20]: Every function in Python returns at least None.
It is better to do one of the following:
- Return True on success and False on fail
- Return nothing but raise an exception on fail

**Review by maxkuzn — CHANGES_REQUESTED** [review]

File: tests.py [hunk 3]
```diff
@@ -7,7 +7,34 @@ def test_remove():
...
 
 def test_set():
-    pass
+    st = Storage({'a': 1, 'b': 2})
+    
+    #change value
+    key = 'a'
+    val = st.get(key)
+    val1 = 10
+    st.set(key,val1)
```
Comment (maxkuzn) [inline tests.py:17]: Check that set returns `True`

File: tests.py [hunk 4]
```diff
@@ -7,7 +7,34 @@ def test_remove():
...
+    res = st.get(key)
+    assert res == val1, "Value for the key {} is not equal to expected".format(key)
+    
+    #text as value
+    key = 'b'
+    val = 'test'
+    st.set(key,val)
```
Comment (maxkuzn) [inline tests.py:25]: Check that set returns `True`

File: tests.py [hunk 5]
```diff
@@ -7,7 +7,34 @@ def test_remove():
...
+    res = st.get(key)
+    assert res == val, "Value for the key {} is not equal to expected".format(key)
+    
+    #inexistent key
+    key = 'c'
+    val = st.get(key)
+    res = st.set(key,val)
```
Comment (maxkuzn) [inline tests.py:32]: Check that set returns `False`

File: tests.py [hunk 6]
```diff
@@ -7,7 +7,34 @@ def test_remove():
...
+    assert val is None, "Value for an unexisting key is not None" 
+    
+    #numerical key
+    key = 4
+    val = st.get(key)
+    res = st.set(key,val)
```
Comment (maxkuzn) [inline tests.py:39]: Check that set returns `False`

**Review by maxkuzn — CHANGES_REQUESTED** [review]

**Pull request merged**